synapse/pipeline.yml

```diff
@@ -267,17 +280,38 @@ steps:
             - /tmp/download-${BUILDKITE_BUILD_ID}/docker-compose.yaml
       - artifacts#v1.3.0:
           upload: [ "_trial_temp/*/*.log" ]
-#      - matrix-org/coveralls#v1.0:
-#          parallel: "true"
     retry: *retry_setup
 
   ################################################################################
   #
   # Sytest
   #
+  # Our tests have three dimensions:
+  #   1. Topology (Monolith, Workers, Workers w/ Redis)
+  #   2. Database (SQLite or PostgreSQL)
+  #   3. Python Version
+  #
+  # We mostly care about testing each topology.
+  # To vary the Python and Postgres versions, we use Docker images which are based
+  # on an assortment of Debian releases.
+  #   - "py35" is Debian Stretch (Python 3.5, Postgres 9.6)
+  #   - "py37" is Debian Buster (Python 3.7, Postgres 11)
+  #   - "py38" is Debian Bullseye (Python 3.9, Postgres 13)
```

Comment:
probably worth noting the debian release numbers here, since we refer to them as 8/9/10 below.

Reply:
Done in ed3a4ad48c26dc8ac965f2026653958eea87a750